packages/go/dawgs/drivers/pg/query/sql/schema_up.sql

```diff
@@ -333,93 +333,6 @@ $$
   parallel safe
   strict;
 
--- Graph helper functions
```

Comment:
Removing these helper functions seems fine from what I can tell. Please confirm this was intentional

Reply:
Can confirm. These functions, while helpful during early development, only represent migration snags now and were removed to simplify the DB schema.